app/enterprise/0.33-x/admin-gui/configuration/authentication.md

```diff
@@ -21,79 +21,147 @@ chapter: 3
 
 ## Enable Authentication
 
-First, we will configure the Admin GUI using [Key Authentication](https://getkong.org/plugins/key-authentication). 
-First, we should add an Administrator entity using the Admin API:
+There are a couple of pieces of information that are important to understand when using authentication for the Admin GUI:
+
+- The Admin GUI is a javascript application that runs in the browser, and makes calls to the Admin API
+- For locking down the GUI, a private endpoint is configured on the proxy side, and an auth plugin is enabled
+- The Admin GUI javascript application uses this proxy side endpoint to talk to the Admin API
+- Two pieces of authentication need to be enabled in order for the Admin GUI to talk to the Admin API in this setup
+- The "Admin user" is a combination of a rbac user and a hidden consumer
```

Comment:
suggestion: instead of `hidden`, maybe we should be explicit:

```and a consumer of type 'admin' that only appears on the `/admins` endpoint and is filtered out from `/consumers` endpoints. Admins as consumers allows Kong to use plugins on an admin to authenticate.```